tests/executables_src/testMapFileParser.cpp

```diff
@@ -139,7 +138,7 @@ BOOST_AUTO_TEST_CASE(testGoodMappFileParse) {
   ChimeraTK::MapFileParser map_file_parser;
   auto [regcat, mdcat] = map_file_parser.parse("goodMapFile.map");
 
-  BOOST_CHECK_EQUAL(regcat.getNumberOfRegisters(), 22);
+  BOOST_CHECK_EQUAL(regcat.getNumberOfRegisters(), 23);
```

Comment:
I was wondering if this is a useful test at all. It always fails when you modify the goodMapFile. The test is useful because getNumberOfRegisters() is public API. But we might want a dedicated test, not run it parasitically on whatever else the "goodMapFile" is testing.